packagingbuild/bionic/Dockerfile

```diff
@@ -41,22 +41,21 @@ RUN echo -e "#!/bin/sh\nexit 101\n" > /usr/sbin/policy-rc.d && \
 # install python development
 RUN apt-get update && \
     apt-get -y install build-essential \
-        python python-virtualenv \
-        python3 python3-virtualenv python3-pip
+        python3-dev python3
 
 RUN apt-get update && \
     apt-get -y install \
         devscripts debhelper dh-make libldap2-dev libsasl2-dev && apt-get clean
 
 # Install fresh pip and co
-RUN curl https://bootstrap.pypa.io/get-pip.py | python3 - virtualenv==16.6.0 pip==20.0.2 wheel setuptools cryptography; \
-      pip3 install --upgrade requests[security] && rm -rf /root/.cache
+RUN curl https://bootstrap.pypa.io/get-pip.py | python3.6 - virtualenv==16.6.0 pip==20.0.2 wheel setuptools cryptography; \
+      pip3.6 install --upgrade requests[security] && rm -rf /root/.cache
 
 # We use our dh-virtualenv version, since it fixes shebangd lines rewrites
 RUN apt-get -y install \
-        python-setuptools python-mock && \
+        python-virtualenv python-setuptools python-mock && \
```

Comment:
Curious - why the move of python-virtualenv to here from earlier?

Reply:
That was done to make Bionic Dockerfile more similar to the Xenial one.

In fact, first I made more changes so those Dockerfiles were almost the same (also added ppa for Python so we install same versions - right now on Xenial we install Python 3.6.12 and on Bionic we install Python 3.6.9, etc.), but then I decided to remove those changes to keep the diff smaller.

If people feel opinionated about it, I can add them back (they work fine as well).